cms/templates/widgets/_ui-dnd-indicator-after.html

```diff
@@ -1,2 +1,2 @@
 <%page expression_filter="h"/>
-<span class="draggable-drop-indicator draggable-drop-indicator-after"><i class="icon fa fa-caret-right"></i></span>
+<span class="draggable-drop-indicator draggable-drop-indicator-after"><span class="icon fa fa-caret-right" aria-hidden="true"></span></span>
```

Comment:
I believe we have a new DnD component and are moving to deprecate the old one, which is not accessible.